R/RcppExports.R

```diff
@@ -1,53 +1,53 @@
 # Generated by using Rcpp::compileAttributes() -> do not edit by hand
 # Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
-#' @title Calculate final epidemic size with RcppEigen backend
+#' @title Iterative solver for final size.
 #'
-#' @description This function calculates final epidemic size using SIR model
-#' for a heterogeneously mixing population.
-#'
-#' @param r0  Basic reproduction number.
-#' @param contact_matrix  Social contact matrix. Entry $mm_{ij}$ gives average
-#' number of contacts in group $i$ reported by participants in group j
-#' @param demography_vector  Demography vector. Entry $pp_{i}$ gives proportion
-#' of total population in group $i$ (model will normalise if needed)
-#' @param prop_initial_infected Proportion of all age groups that is initially
-#' infected. May be a single number, or a vector of proportions infected.
-#' If a vector, must be the same length as the demography vector, otherwise the
-#' vector will be recycled. Default value is 0.001 for all groups.
-#' @param prop_suscep  Proportion of each group susceptible. May be a single
-#' numeric value or a numeric vector of the same length as the demography
-#' vector.
-#' @param iterations Number of solver iterations
+#' @param contact_matrix Social contact matrix. Entry \eqn{mm_{ij}} gives
+#' average number of contacts in group \eqn{i} reported by participants in
+#' group \eqn{j}.
+#' @param demography_vector Demography vector. Entry \eqn{pp_{i}} gives
+#' proportion of total population in group \eqn{i}
+#' (model will normalise if needed).
+#' @param susceptibility A matrix giving the susceptibility of individuals in
+#' demographic group \eqn{i} and risk group \eqn{j}.
+#' @param iterations Number of solver iterations. Defaults to 1,000.
+#' @param tolerance Solver error tolerance.
```

Comment:
More details here would be useful. What does "solver error tolerance" mean in practice? As a user, how my results will be impacted if I change it.

Reply:
Added some basic description in https://github.com/epiverse-trace/finalsize/pull/73/commits/17e198a3cb896016840cbb759d9e0a03be6ae037 with operation information that larger values may lead to inaccurate final size.